.github/workflows/linting.yml

```
@@ -0,0 +1,43 @@
+name: Linting
+
+on:
+  push:
+    branches: [ sj/ga-migrate-linting ]
```

Updated the branch name. Didn't we decide to run this on pull request rather than push?
```suggestion
on:
  push:
    branches: [ development ]
```